tests/core/pyspec/eth2spec/test/eip7251/block_processing/test_process_consolidation.py

```diff
@@ -145,13 +144,11 @@ def test_consolidation_churn_limit_balance(spec, state):
 def test_consolidation_balance_larger_than_churn_limit(spec, state):
     # This state has 256 validators each with 32 ETH in MINIMAL preset, 128 ETH consolidation churn
     consolidation_churn_limit = spec.get_consolidation_churn_limit(state)
-    # Set the consolidation balance to consume equal to churn limit
-    state.consolidation_balance_to_consume = consolidation_churn_limit
     current_epoch = spec.get_current_epoch(state)
 
     source_index = spec.get_active_validator_indices(state, current_epoch)[0]
     # Set source balance higher than consolidation churn limit
-    state.balances[source_index] = consolidation_churn_limit + 1
+    state.validators[source_index].effective_balance = 2 * consolidation_churn_limit
```

Comment:
```suggestion
    state.validators[source_index].effective_balance = consolidation_churn_limit + spec.EFFECTIVE_BALANCE_INCREMENT
```

I'd maybe do this instead, just to make it easier to interpret what happens. Here one sees the effective balance being 2x the churn limit, and it can be confusing why then `expected_exit_epoch` only gets a +1 instead of +2